content/bluetooth.md

```diff
@@ -72,44 +99,135 @@ This process lowers the sound quality of the stream when in HSP/HFP mode, so aud
 
 Bluetooth issues can be troubleshooted in several ways.  The first thing to check is toggling airplane mode which will sometimes get Bluetooth functioning again.  Next, make sure Bluetooth is enabled in the top bar, or in the <u>Bluetooth</u> system settings.
 
```

Comment:
After the airplane mode, I suggest adding a new line and a screenshot for this section:
Confirm Bluetooth is enabled in the top bar, or in the <u>Bluetooth</u> system settings.